importer/SeBbrSv.py

```diff
@@ -340,13 +354,22 @@ def check_wd_item(self):
         Detect BBR conflict in associated WD item.
 
         Get any P1260 of associated WD item.
+
+        If it's 1 BBR building link AND the compound consists of 1 building,
+        and the link is the same as the link to that building,
+        keep the item association.
+
         If it's a BBR ID that's different
         from the one in the data object,
         remove the association. Data will be uploaded
         to a newly created WD item instead.
         """
         bbr_url_on_wd = self.get_wditems_bbr_links()
+        parts_of_compound = self.get_has_parts()
         if bbr_url_on_wd:
+            if len(parts_of_compound) == 1 and len(bbr_url_on_wd) == 1:
```

Comment:
There is one case not handled.
`len(bbr_url_on_wd) == 2` and `len(parts_of_compound) == 1` and the two ids are that of the compound and that of the building.

Easiest structure might be something like the below but I'm open for others:
```python
if len(bbr_url_on_wd) == 0:
    return
elif len(bbr_url_on_wd) == 1:
    if bbr_url_on_wd[0] == self.bbr_link:
        return
    elif len(parts_of_compound) == 1 and parts_of_compound[0] == bbr_url_on_wd[0]
        return
elif len(bbr_url_on_wd) == 2 and len(parts_of_compound) == 1:
    if all(entry in bbr_url_on_wd for entry in [parts_of_compound[0], self.bbr_link]):
        return
self.wd_item["wd-item"] = None
```